src/Forms/Shared/Samples/Map/DisplayOverviewMap/readme.metadata.json

```diff
@@ -19,11 +19,11 @@
     ],
     "nuget_packages": {
         "Esri.ArcGISRuntime": "100.13.0",
+        "Esri.ArcGISRuntime.Toolkit.Xamarin.Forms": "100.13.0",
         "Esri.ArcGISRuntime.UWP": "100.13.0",
         "Esri.ArcGISRuntime.Xamarin.Android": "100.13.0",
         "Esri.ArcGISRuntime.Xamarin.Forms": "100.13.0",
-        "Esri.ArcGISRuntime.Xamarin.iOS": "100.13.0",
-        "Esri.ArcGISRuntime.Toolkit.Xamarin.Forms": "100.13.0"
+        "Esri.ArcGISRuntime.Xamarin.iOS": "100.13.0"
```

Comment:
Another unrelated change - maybe the script needs to be updated in a separate PR?